src/components/ViewVariableValue/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { Fragment } from "react";
+import Modal from "components/Modal";
+import Button from "components/Button";
+import Table from "react-bootstrap/Table";
```

Comment:
Can we do this table similar to the ssh keys - i.e. not requiring bootstrap as well?